README.md

```diff
@@ -429,19 +429,19 @@ The `adoptedStyles` property is key, as this will efficiently allow every style
 
 ## Compatibility
 
-### Typescript
+### TypeScript
 
-This library relies on the new decorators, so you \*_must not_ have `experimentalDecorators` enabled in your `tsconfing.json`. We recommend having `target: "es2015"` or later set as well.
+This library relies on the current experimental TypeScript implementation of the decorator syntax, so you \*_must not_ have `experimentalDecorators` enabled in your `tsconfing.json`. We recommend having `target: "es2015"` or later set as well.
```

Comment:
This is slightly confusing. Perhaps I need to expound on this topic a bit more. What happened is that TypeScript implemented decorators behind the `experimentalDecorators` flag, then ECMAScript actually made a proposal for decorators which quite substantially deviates from what TypeScript had implemented. 

This library is using the ECMAScript standard decorators, which is why the `experimentalDecorators` flag needs to be turned off.

Reply:
Oh right, I misread `must not` as `must`, which means that my text is flat out wrong. I will review it one more time in a follow-up.